tests/foreman/api/test_permission.py

```diff
@@ -38,28 +38,41 @@ def create_permissions(self, class_target_sat):
         # workaround for setting class variables
         cls = type(self)
         cls.permissions = PERMISSIONS.copy()
-        if class_target_sat.is_upstream:
-            cls.permissions[None].extend(cls.permissions.pop('DiscoveryRule'))
-            cls.permissions[None].remove('app_root')
-            cls.permissions[None].remove('attachments')
-            cls.permissions[None].remove('configuration')
-            cls.permissions[None].remove('logs')
-            cls.permissions[None].remove('view_cases')
-            cls.permissions[None].remove('view_log_viewer')
-
-        result = class_target_sat.execute('rpm -qa | grep rubygem-foreman_openscap')
-        if result.status != 0:
+        if not class_target_sat.is_upstream:
+            cls.permissions.pop('LookupValue')
```

Comment:
This doesn't seem right. On my Satellite (stream and 6.15):
```
# hammer filter available-permissions | grep -i lookup
213 | view_lookup_values                              | LookupValue                    
214 | create_lookup_values                            | LookupValue                    
215 | edit_lookup_values                              | LookupValue                    
216 | destroy_lookup_values                           | LookupValue
```

Reply:
Good catch. I have been looking through the diff but I don;t know why I thought this would not be present on satellite ...